Makefile

```diff
@@ -1,13 +1,11 @@
 test:
 	pytest --cov-report term-missing --cov=rich tests/ -vv
-format-check:
-	black --check .
+lint:
+	pre-commit run -a
 format:
-	black .
+	pre-commit run black -a
 typecheck:
-	mypy -p rich --ignore-missing-imports --warn-unreachable
-typecheck-report:
```

Comment:
I'd like to be able to generate the mypy report as needed.

Reply:
I put it back.